### ISSUE TITLE
Channel was forced closed now i can't open new channels from other nodes. 

### ISSUE THREAD

**Issue by corbimos** [ElementsProject/lightning#2129]
I have two nodes, one is LND and the other is c-lightning. Both are BTCpay servers.

<strike>I also have an e-clair lightning wallet.</strike> *I still have it but it is connecting fine now.

https://github.com/lightningnetwork/lnd/issues/2250

I opened the above issue on the lnd side but it seems my c-lightning is disconnecting from other new channels.

<strike>My eclair wallet shows 'channel funding error' when attempting to open a channel as well. Screenshot of channel details attached. </strike>

I can still make payments with the active channels. But i cannot open new ones from the other side. 

I need the the funds to be on the lnd side. I did have a similar issue in the past and updating both the lnd and c-lightning nodes fixed the issue. 

[lnd.log](https://github.com/ElementsProject/lightning/files/2638189/lnd.log)
[c-lightning.log](https://github.com/ElementsProject/lightning/files/2638231/c-lightning.log)

### `getinfo` output

{
  "id": "036dc6c5fc1a16171f182f48794cf3833654e8018cce0bde540854b19854448f5f", 
  "alias": "Corbimos", 
  "color": "036dc6", 
  "address": [
    {
      "type": "ipv4", 
      "address": "23.100.30.211", 
      "port": 9735
    }
  ], 
  "binding": [
    {
      "type": "ipv4", 
      "address": "0.0.0.0", 
      "port": 9735
    }
  ], 
  "version": "85e4d9e-modded", 
  "blockheight": 552346, 
  "network": "bitcoin", 
  "msatoshi_fees_collected": 7623
}

**Comment by corbimos** [comment]
Today i was able to connect to my lnd node from c-lightning for the first time since the original channel was closed, i'm not entirely sure what has changed. 

I was able to open a channel with funds on the c-lightning side,  I will just create an invoice and push them to the lnd side to achieve my goal.

Again, i'm not understanding why it forced closed to begin with, but things seem to jiving now... my least favorite way to get a solution. 

Closing the issue for now, will update if things change.
